packages/react-dom/package.json

```diff
@@ -34,6 +34,7 @@
     "server.node.js",
     "test-utils.js",
     "unstable_testing.js",
+    "ReactDOMDispatcher.js",
```

Comment:
We'll need to make sure before merge that we don't accidentally actually include this file on npm. Could be `unstable_` something unless there's already some final name.

Reply:
Talked with Seb already and we are going to route through react dom internals until 19 when we can do more drastic separation of isomorphic / client / server methods. But yeah he mentioned the same thing about using a script to suppress this before publish which makes sense for when we do this in a major